apps/desktop-wallet/src/pages/NewWallet/CreateWalletPage.tsx

```diff
@@ -90,36 +90,41 @@ const CreateWalletPage = ({ isRestoring = false }: { isRestoring?: boolean }) =>
   }
 
   const handleNextButtonClick = async () => {
+    sendAnalytics({ event: 'Creating wallet: Creating password: Clicked next' })
+
     try {
       saveNewWallet({ walletName, encrypted: await encryptMnemonic(mnemonic, password) })
       resetCachedMnemonic()
 
       if (isRestoring) {
         discoverAndSaveUsedAddresses({ skipIndexes: [0], enableLoading: false })
-        posthog.capture('New wallet imported', { wallet_name_length: walletName.length })
+        sendAnalytics({ event: 'New wallet imported', props: { wallet_name_length: walletName.length } })
       } else {
-        posthog.capture('New wallet created', { wallet_name_length: walletName.length })
+        sendAnalytics({ event: 'New wallet created', props: { wallet_name_length: walletName.length } })
       }
 
       onButtonNext()
-    } catch (e) {
-      console.error(e)
-
+    } catch (error) {
       if (isRestoring) {
-        dispatch(walletCreationFailed(getHumanReadableError(e, t('Error while importing wallet'))))
-        posthog.capture('Error', { message: 'Could not import wallet' })
+        dispatch(walletCreationFailed(getHumanReadableError(error, t('Error while importing wallet'))))
+        sendAnalytics({ type: 'error', error, message: 'Could not import wallet', isSensitive: true })
```

Comment:
I'll remove the error from here because even though we clean up potential mnemonic leak, we might leak wallet name. Same as below.

Reply:
@mvaivre  I'll add some more granular error reporting in `saveNewWallet` so that we can safely report the `error` here.